src/z-autocomplete.ts

```diff
@@ -64,12 +62,17 @@ export class ZAutocomplete extends LitElement {
     @property()
     set value(val: any) {
         this._value = val;
-        this._clearOptions();
 
+        // udpate input value first
         const option = this.dataToOption(val);
-        this._inputEl.value = option?.inputValue
-            ?? (typeof option?.label === 'string' ? option?.label : '')
-            ?? '';
+        if (option?.inputValue) this._inputEl.value = option.inputValue;
+        else if (typeof option?.label === 'string') this._inputEl.value = option.label;
+        else if (option?.label instanceof HTMLElement) this._inputEl.value = option.label.textContent ?? '';
```

Comment:
Si jamais il y avait une alternative en dessous ça serait plus safe de mettre

```suggestion
        else if (option?.label instanceof HTMLElement && option.label.textContent) this._inputEl.value = option.label.textContent;
```

Reply:
on est rendu sur du pinaillage là ^^

on ne fait pas la même chose pour 

`if (typeof option?.label === 'string') this._inputEl.value = option.label;`

donc perso je le ferais partout ou nulpart

idem pour `option.inputValue`

Reply:
le `?? ''` sert surtout pour l'histoire de typage

si on veut contrôler chaque donnée de l'implémentation on est pas rendu à mon avis ^^

Reply:
okok, c'est pas méga important, y a peu de chances que 'lutilisateur n'affiche aucun texte dans son élément de toute façon